package.json

```diff
@@ -141,5 +141,8 @@
     "webpack-dev-server": "^4.9.0",
     "webpack-stats-plugin": "^1.0.3"
   },
+  "resolutions": {
+    "@types/react": "^16.14.26"
```

Comment:
This little gem tells yarn to only install 1 copy of these types.  Without it, there was a transient-dependency that was including `@types/react: 18` and was messing up typechecks in various places.